docs/transactions/iti41.md

```diff
@@ -2,40 +2,170 @@
+#### MTP
+* Treatment duration (defined in [dosage](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-dosage.html)) should be within the period of validity of the document (defined by [`DocumentEntry.serviceStatTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.19) and [`DocumentEntry.serviceStopTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.20) metadata).
+#### PRE
+* Effective time of [`documentationOf`](#FIXME) and [`serviceEvent`](#FIXME) has to be aligned with [`DocumentEntry.serviceStatTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.19) and [`DocumentEntry.serviceStopTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.20).
+* Treatment duration (defined in [dosage](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-dosage.html)) should be within the period of validity of the document (defined by [`DocumentEntry.serviceStatTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.19) and [`DocumentEntry.serviceStopTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.20) metadata).
+* Each PRE item should refer to an MTP item that must be a [available](#FIXME) [approved](#FIXME) [valid](#FIXME) [active](#FIXME) and for the same patient
+    * Not blocking issues should not prevent publication but should be notified to service administration to address the resulting ad-hoc issue    
+* [Validity of document](#FIXME) has to be after the [date of the PRE document](#FIXME) and the [start date of duratio](#FIXME)n of treatment of parent MTP 
+* [Code of medication](#FIXME) should match the code of medication of referenced MTP.      
+* If [lot number](#FIXME) is specified in referenced MTP, the PRE item mast match it.
+#### PADV
+##### PADV against a [provisional PRE](#FIXME)
+* Medication cannot be changed.
+* Dosage instructions cannot be changed.
+* Substitution cannot be disallowed if the existing dispense includes a substitution.
+* Observation code `REFUSE` cannot be issued.
+#####  PADV `CANCEL`
+* Not allowed against DIS or other PADV documents.
+
+#### DIS
+* Dispense [status](#FIXME) has to be coherent with previous DIS documents for the same prescription.
+* If [lot number](#FIXME) is specified in parent MTP or PRE, lot number must match.
+##### DIS against PRE:
+* Medication cannot be different if substitution is not allowed by PRE. 
+    * Verification Is performed on the [GTIN](#FIXME) of the medication (not of the package). 
+    * If a substitution other than `none` is specified, no validation is performed.
+* Dispense cannot occur after the [end of validity of the PRE](#FIXME) document.
+* Dispense cannot occur after the [end of the treatment](#FIXME) if specified in the [prescription item](#FIXME).
+* If referenced PRE is not [provisional](#FIXME), it should be [dispensable](#FIXME).
+* List of active ingredients should be the same than the one in the [parent PRE](#FIXME).
```

Comment:
Similar to the medication one... the list of ingredients is given in `CHEMEDEPRMedication.ingredient` of which `CHEMEDEPRMedication.ingredient.isActive` is true.

Reply:
This is not checked atm. Should it be implemented, it will result in a warning.
Note this check should be done even if substitution is allowed.